Connection.php

```diff
@@ -287,12 +287,21 @@ protected function _run(HoaSocket\Node $node)
                             break;
                         }
 
-                        $this->getListener()->fire(
-                            'message',
-                            new Event\Bucket([
-                                'message' => $frame['message']
-                            ])
-                        );
+                        try {
```

Comment:
My thought was to only catch `Exception\Group<Socket\Exception\BrokenPipe>` to let the upper catch block work in others case.